content/news/release-20170527.yaml

```diff
@@ -0,0 +1,7 @@
+---
+kind: article
+title: "Release 20170527 hotfix now available"
+author: "Paul Chote"
+created_at: 2017-05-23 20:00 +0000
+disqus_id: "release-20170421"
```

Comment:
Note: this is intentional.  I don't see any need to reset the comments for these minor releases.